### DISCUSSION TITLE
How to show a markdown table format content in react-markdown ?

### DISCUSSION THREAD

**Discussion by iamvermajayant** [remarkjs/remark#1295] [Q&A]
| Metric Category | Metric Detail | Description | |-----------------|--------------------------|--------------------------------------------------------------------------------------------------| | Complexity | Skill Depth Needed | High: The data indicates a need for expertise in areas such as contract management, strategic planning, and project performance assessment. | | | Tech Stack | Diverse: The innovation engagements involve a variety of technologies, including data analysis tools, project management software, and digital communication platforms. | | Impact | Efficiency Improvements | Notable: The innovation engagements have led to improvements in areas such as contract renewal processes, project performance, and supplier interactions. | | | Cost Savings | Potential: The strategic decisions guided by these innovation engagements could lead to cost savings through more efficient processes and better contract management. | | | Revenue Growth | Evident: The innovation engagements have a clear focus on areas that can drive revenue growth, such as improving project performance and enhancing supplier interactions. | | | Enhanced User Experiences| Significant: The focus on areas such as hot topics and supplier interactions suggests a commitment to improving user experiences. |

This is the response that I am getting from the FAST api, and I am rendering this into react-markdown it is not coming into table format 
Is there any way to do it

**Comment by remcohaszing** [comment]
Your response doesn’t contain any newlines. It’s not rendered as a table on GitHub either.
